### DISCUSSION TITLE
Turn on Auto suggestion

### DISCUSSION THREAD

**Discussion by tachibana-shin** [starship/starship#2913] [Q&A] [answered]
Hi,

I accidentally turned off auto sub suggest suggestion. How do i reactivate it?

Screenshot

![image](https://user-images.githubusercontent.com/45375496/126859845-f9adbaea-8cd5-4591-8def-6457f0232b9b.png)

**Comment by matchai** [comment, marked as answer]
Starship has no completion or suggestion features. Please read this FAQ entry for more details:
https://starship.rs/faq/#how-do-i-get-command-completion-as-shown-in-the-demo-gif

**Comment by execrc** [comment]
You can use `fish` or `zsh` shell for getting auto suggestions in terminal.
